ui.frontend/src/GuideBridge.js

```diff
@@ -247,6 +255,42 @@ export default class GuideBridge {
         //TODO: implement it later. NO-OP for now.
     }
 
+    /**
+     *
+     * @param eventName name of the event to trigger on GuideBridge
+     * @param eventPayload
+     * @param formContainerPath if no argument passed, use any form container
+     */
+    trigger(eventName, eventPayload, formContainerPath) {
+        let formContainer;
+        if (formContainerPath) {
+            formContainer = this.#formContainerViewMap[formContainerPath];
```

Comment:
we should have a generic utility to get the formcontainerpath. it should have a fallback: if the variable is not initialise, return the first of the containers. without explicitly defining getRandomFormContainerPath anywhere in the code.